docs/acceptance-mobile.md

```diff
@@ -0,0 +1,83 @@
+- [ ] It should allow users to log into or create a Firefox account
+- [ ] It should allow users to select and send one file at a time
+- [ ] It should limit the sendable file size to 1GB
+- [ ] It should allow users to set an expiration time of 5 minutes, 1 hour, or 24 hours
+- [ ] It should allow users to set an download count of 1, 10 or 20 downloads
+
+### Authenticated Users
+- [ ] It should indicate that the user is signed in via Firefox Account
+- [ ] It should allow the user to sign out
+- [ ] It should allow users to select and send multiple files in one URL
+- [ ] It should limit users to sending 2.5 GB per Send.
+- [ ] It should allow users to extend Send times up to 1 Week
+- [ ] It should allow users to extend Send download counts up to 200 times
+
+### Cards
+- [ ] It should display the name of the sent file/files
+- [ ] It should display the time remaining before expiration
+- [ ] It should display the number of downloads remaining before expiration
+- [ ] It should have a button that lets the user copy the send link to their clipboard
+- [ ] It should show a preview icon (not a thumbnail) that has some relationship to the file types or content being sent* (see 5.1 in spec)
+- [ ] It should have an overflow (meatball) menu that when triggered, gives the user share or delete buttons
+- [ ] While encrypting / pushing to server, it should display a progress meter and a cancel button
+- [ ] For authenticated users, it should be expandable to display all files in a send (5.1.1)
+- [ ] If user cancels Send, or Upload fails, it should display a warning in the card
+- [ ] It should display expired Sends below current sends with their UI greyed out and an expiration warning for 24 hours after expiration
+- [ ] It should remove expired cards from display after 24 hours
+- [ ] It should let users permanently delete records expired sends
+- [ ] It should display a visual indicator when a Send is password protected
+- [ ] It should allow the user to share via a native Android share sheet
+
+### General/other
+- [ ] It should allow users to set passwords to protect their Sends
+- [ ] It should warn users when they are trying to upload files larger than their share limit
+
+### Stretch
+- [ ] It should allow users to use the photo gallery to create Send links
+- [ ] It should allow users to use their camera to create Send links
+- [ ] It should allow me to create Send links through intents from other apps
+- [ ] It should trigger a notification when a share link expires
+- [ ] It should allow users to opt into notifications when their link is downloaded
+
+## Annotations on Mobile Spec
+This document tracks differences between the UX spec for Firefox Send and the intended MVP.
+
+[Spec Link](https://mozilla.invisionapp.com/share/GNN6KKOQ5XS)
+
+* 1.1: Spec describes toolbar which may not be possible given the application framework we're using. In particular, issues with the spec include the color, logo and different font weights may be at issue.
```

Comment:
@johngruen  So what would be the suggestion here? Show only "Firefox Send" with the same font-weight without the icon?

Reply:
@youwenliang We just can't really control it, it's controlled by the OS. I can change the text to say Firefox Send, but here's what it looks like now.
<img width="243" alt="screen shot 2018-08-24 at 8 09 45 am" src="https://user-images.githubusercontent.com/46048/44583911-34951d80-a775-11e8-8192-9e04ae6c6379.png">

Reply:
@fzzzy can we seet the color?